bitacross-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -177,6 +176,13 @@ where
 		}))
 	});
 
+	io.add_sync_method("state_getScheduledEnclaveList", move |_: Params| {
+		debug!("worker_api_direct rpc was called: state_getScheduledEnclaveList");
+		let value = format!("{:?}", &GLOBAL_SCHEDULED_ENCLAVE.registry.read());
```

Comment:
This results in something like this:
`Ok({0: [177, 155, 22, 22, 112, 239, 100, 252, 239, 19, 10, 78, 170, 5, 181, 229, 13, 234, 219, 91, 29, 160, 23, 147, 133, 219, 42, 14, 151, 253, 19, 134]})`

1. You should return only value contained in `Result` not `Result` itself.
2. In my opinion value should be serialized directly, not formated to string then serialized.
3. Can we have an integration test for this ?